1.css

```diff
@@ -0,0 +1,298 @@
+
+@import url('https://fonts.googleapis.com/css?family=Montserrat');
+
+* {
+    margin: 0px;
+    padding: 0px;
+}
+body {
+    height: 100%;
+  }
+
+p{
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs